datacenterlight/templates/datacenterlight/base.html

```diff
@@ -1,4 +1,4 @@
-{% load staticfiles i18n%}
+{% load staticfiles i18n cms_tags sekizai_tags %}
```

Comment:
Yes, we are using base.html in non-cms pages. Precisely in index, whydatacenterlight, success pages.
The cms_tags (including cms_toolbar) are not rendered in non-cms pages.